src/core/main.c

```diff
@@ -379,6 +427,14 @@ static int parse_proc_cmdline_item(const char *key, const char *value, void *dat
                 else
                         arg_crash_reboot = r;
 
+        } else if (proc_cmdline_key_streq(key, "systemd.crash_self_healing")) {
+
+                r = value ? parse_boolean(value) : true;
+                if (r < 0)
+                        log_warning_errno(r, "Failed to parse crash self_healing switch %s, ignoring: %m", value);
```

Comment:
I see that this copies what the surrounding code does, but the surrounding code is not good ;)
Please use the actual swtich name:
```C
log_warning_errno(r, "Failed to parse %s=%s, ignoring: %m", key, value);
```